packages/manager/apps/dedicated/client/app/index.js

```diff
@@ -6,14 +6,19 @@ import {
 } from '@ovh-ux/manager-preloader';
 import { bootstrapApplication } from '@ovh-ux/manager-core';
 import { Environment } from '@ovh-ux/manager-config';
+import { User } from '@ovh-ux/manager-models';
 
-attachPreloader(Environment.getUserLanguage());
-
-bootstrapApplication('dedicated').then(({ region, message }) => {
+bootstrapApplication('dedicated').then(({ message, region, user }) => {
   if (message) {
     displayMessage(message, Environment.getUserLanguage());
   }
-
+  const currentUser = new User(user);
+  attachPreloader(
+    Environment.getUserLanguage(),
+    currentUser.isTrusted
+      ? '<ovh-manager-trusted-nic-label></ovh-manager-trusted-nic-label>'
```

Comment:
You can't use an AngularJS component in preloader, as when the preloader is displayed Angular is not yet initialised. 
It will work with a delay...

Reply:
This change has been removed, after confirmation with the PU. This is not needed.